crates/uv-virtualenv/src/virtualenv.rs

```diff
@@ -184,91 +227,100 @@ pub(crate) fn create(
         }
     }
 
-    // No symlinking on Windows, at least not on a regular non-dev non-admin Windows install.
+    // On Windows, we use trampolines that point to an executable target. For standalone
+    // interpreters, this target path includes a minor version junction to enable
+    // transparent upgrades.
     if cfg!(windows) {
-        copy_launcher_windows(
-            WindowsExecutable::Python,
-            interpreter,
-            &base_python,
-            &scripts,
-            python_home,
-        )?;
-
-        if interpreter.markers().implementation_name() == "graalpy" {
-            copy_launcher_windows(
-                WindowsExecutable::GraalPy,
-                interpreter,
-                &base_python,
-                &scripts,
-                python_home,
-            )?;
-            copy_launcher_windows(
-                WindowsExecutable::PythonMajor,
-                interpreter,
-                &base_python,
-                &scripts,
-                python_home,
-            )?;
+        if interpreter.is_standalone() {
+            let target = scripts.join(WindowsExecutable::Python.exe(interpreter));
+            create_bin_link(target.as_path(), executable_target.clone()).map_err(Error::Python)?;
```

Comment:
I think the name of this function is way too general now that it's unattached from a `ManagedPythonInstallation`. Previously, it was quite specifically "create a `bin/` link to this managed Python interpreter". Now it seems to be used for more? We should probably rename it for clarity, and update the doc?

Reply:
Updated in #14029